src/utils/actions.js

```diff
@@ -100,90 +100,56 @@ const actions = [
     value: 'unlockField',
     label: 'Unlock a specific field',
     group: 'selection',
-    call: ({ engineApp, field }) => async () => {
+    getActionCall: ({ engineApp, field }) => async () => {
       if (field) {
         const fieldObj = await engineApp.getField(field);
         await fieldObj.unlock();
       }
     },
     requiredInput: ['field'],
   },
-  {
-    value: 'unlockAllAndClearAll',
-    label: 'Unlock all and clear all',
-    group: 'selection',
-    call: ({ engineApp }) => async () => {
-      await engineApp.unlockAll();
-      await engineApp.clearAll();
-    },
-    requiredInput: [],
-  },
-  {
-    value: 'selectField',
-    label: 'Select a value in a field',
-    group: 'selection',
-    call: ({ engineApp, field, value }) => async () => {
-      if (field && value) {
-        const fieldObj = await engineApp.getField(field);
-        await fieldObj.select(value);
-      }
-    },
-    requiredInput: ['field', 'value'],
-  },
   {
     value: 'selectAll',
     label: 'Select all values in a field',
     group: 'selection',
-    call: ({ engineApp, field }) => async () => {
+    getActionCall: ({ engineApp, field, softLock }) => async () => {
       if (field) {
         const fieldObj = await engineApp.getField(field);
-        await fieldObj.selectAll();
+        await fieldObj.selectAll(softLock);
       }
     },
-    requiredInput: ['field'],
+    requiredInput: ['field', 'softLock'],
   },
   {
     value: 'selectValues',
     label: 'Select multiple values in a field',
```

Comment:
I think we should change this label to 'Select value(s) in a field', since we are removing the single one